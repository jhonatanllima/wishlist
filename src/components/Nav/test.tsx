import { screen } from '@testing-library/react';

import { IoLocationSharp } from 'react-icons/io5';
import { FaHeart, FaPhoneAlt } from 'react-icons/fa';

import { renderWithTheme } from '~/utils/tests/helpers';

import { Nav } from '.';

const menuItems = [
  {
    title: 'Cidade: SÃ£o Paulo',
    icon: <IoLocationSharp />,
  },

  {
    title: 'Central de Atendimento',
    icon: <FaPhoneAlt />,
  },

  {
    title: 'Lista de Desejos',
    icon: <FaHeart />,
    link: '/favoritos',
  },
];

describe('<Nav /> ', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<Nav menuItems={menuItems} />);

    expect(screen.getByRole('nav')).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
