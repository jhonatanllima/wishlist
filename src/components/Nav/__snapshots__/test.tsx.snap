// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< Nav />  should render the heading 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  background: blue;
}

<nav
  class="c0"
>
  <h1>
    Nav
  </h1>
</nav>
`;
